### PR TITLE
docs: update simple exec

### LINKED ISSUE #167: Update simple-executor section
This section is out-of-date and contains wrong information.

We should not add copied code from the core since we risk of having it outdated

### PR DESCRIPTION
**Pull request by Yongxuanzhang**
### All Submissions:

* [ ] Have you followed the [documentation style guide](https://github.com/jina-ai/docs/tree/master/page_templates/style_guide.md)? 

### New Feature Submissions:

1. [x] Have you added your new file to a table of content index? 
2. [ ] Did you use one of the templates in the [page_template](https://github.com/jina-ai/docs/tree/master/page_templates) folder?
3. [x] Is your title in Title Case? 
4. [x] Are your section headers in sentence case?

### CODE REVIEW

File: chapters/simple_exec.md [hunk 1]
```diff
@@ -17,44 +17,36 @@ metas:
   name: clear
 requests:
   on:
-    [SearchRequest, TrainRequest, IndexRequest]:
-      - !ReqPruneDriver {}
+    SearchRequest:
+      - !ExcludeReqQL
```
Comment (JoanFM) [inline chapters/simple_exec.md:25]: are u sure this is the name of a Driver?

Comment (Yongxuanzhang) [reply chapters/simple_exec.md:25]: https://github.com/jina-ai/jina/blob/master/jina/resources/executors._clear.yml

Comment (Yongxuanzhang) [reply chapters/simple_exec.md:25]: I think it's this one?

Comment (Yongxuanzhang) [reply chapters/simple_exec.md:25]: This YAML file is also updating frequently, so we'd better guide the location of these execs

**Comment by Yongxuanzhang** [comment]
This is to update the previous section, should I make it consistent with the template? @FionnD 
The structure seems good enough. 😄 

Preview: https://deploy-preview-177--jina-docs.netlify.app/chapters/simple_exec/

**Review by nan-wang — APPROVED** [review]
LGTM👍

File: chapters/simple_exec.md [hunk 2]
```diff
@@ -1,9 +1,13 @@
 # Built-in Simple Executors and Reserved `uses` in Jina
 
+- [Built-in Simple Executors and Reserved `uses` in Jina](#built-in-simple-executors-and-reserved--uses--in-jina)
```
Comment (bwanglzu) [inline chapters/simple_exec.md]:
```suggestion
- [Built-in Simple Executors and Reserved keyword `uses` in Jina](#built-in-simple-executors-and-reserved--uses--in-jina)
```

**Review by bwanglzu — CHANGES_REQUESTED** [review]
Great work! Left minor comment

**Review by bwanglzu — APPROVED** [review]
LGTM!

File: chapters/simple_exec.md [hunk 3]
```diff
@@ -1,9 +1,13 @@
...
 
+- [Built-in Simple Executors and Reserved keyword `uses` in Jina](#built-in-simple-executors-and-reserved--uses--in-jina)
+  * [What is Simple Executor?](#what-is-simple-executor)
+  * [What are the reserved `uses`?](#what-are-the-reserved-uses)
+  * [How to use built-in Simple Executors?](#how-to-use-built-in-simple-executors)
 
 ## What is Simple Executor?
 
-In Jina we have several built-in some *simple* executors. They are called simple for two reasons:
+In Jina we have several built-in some *simple* executors, which are located in `jina/resources/`. They are called simple for two reasons:
```
Comment (FionnD) [inline chapters/simple_exec.md:10]: In Jina we have several built-in *simple* Executors, 

(Remove some)

File: chapters/simple_exec.md [hunk 4]
```diff
@@ -1,9 +1,13 @@
...
 
 - It only requires a YAML config, no Python code is required;
```
Comment (FionnD) [inline chapters/simple_exec.md:12]: They only require a YAML configuration file. 
They inherit from `BaseExecutor` directly, therefore the logic thus fully relies on the drivers.

File: chapters/simple_exec.md [hunk 5]
```diff
@@ -17,44 +21,36 @@ metas:
...
-    :param needs: list of service names to wait
-    :return: the modified flow
-    """
-    if len(needs) <= 1:
-        raise FlowTopologyError('no need to wait for a single service, need len(needs) > 1')
-    return self.add(name='joiner', uses='_pass', needs=needs, *args, **kwargs)
-
-```
-
+It cleans up request from the request-level protobuf message to reduce the total size of the message. This is often useful when the proceeding Pods require only a signal, not the full message.
```
Comment (FionnD) [inline chapters/simple_exec.md:53]: It cleans up request**s**

File: chapters/simple_exec.md [hunk 6]
```diff
@@ -63,15 +59,19 @@ To help users quickly use these patterns, we reserved the following keywords for
...
-
+| `_merge_chunks` | Merges chunks from requests |
+| `_merge_matches` | Merges matches from requests |
+| `_merge_matches_topk` | Merge the top-k search results at match level |
+| `_merge_eval` | Merge all evaluations into one, grouped by ``doc.id`` |
+| `_merge_root` | Merge results at root level |
 
 ## How to use built-in Simple Executors?
 
 You can directly use this executor by specifying `--uses=_clear`, or use it in `--uses-after` after collecting results from replicas.
```
Comment (FionnD) [inline chapters/simple_exec.md:75]: or use it in `--uses-after` after collecting results from replicas. 
That sentence is not grammatically correct?

**Review by FionnD — COMMENTED** [review]
Hi! 
There are some grammar mistakes, please fix them and re-merge the PR

**Comment by Yongxuanzhang** [comment]
> Hi!
> There are some grammar mistakes, please fix them and re-merge the PR

Sorry about that. I will fix them now

**Pull request merged**